src/main/java/xyz/aadev/generitech/api/util/MachineTier.java

```diff
@@ -38,23 +38,30 @@
 import xyz.aadev.generitech.Reference;
 
 public enum MachineTier implements IStringSerializable {
-    TIER_0("t0"),
-    TIER_1("t1"),
-    TIER_2("t2"),
-    TIER_3("t3");
+    TIER_0("t0", 0),
+    TIER_1("t1", 1),
+    TIER_2("t2", 2),
+    TIER_3("t3", 3);
     // N/A
 
     private static final MachineTier[] META_LOOKUP = new MachineTier[values().length];
+    private static MachineTier[] value;
 
     static {
         for (MachineTier tier : values()) {
             META_LOOKUP[tier.getMeta()] = tier;
         }
     }
 
+    static {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either remove or fill this block of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.aadev.xyz/coding_rules#rule_key=squid%3AS00108)